packages/bap-server/bap-server.master/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+name: "bap-server"
+version: "master"
```

Comment:
are all the `master` versions of the packages normal?

Reply:
> are all the `master` versions of the packages normal?

Nope, the master packages are not a part of the release so I copied them accidentally from our internal opam-repository. 

> Closing until the syntax errors making all the jobs fail is resolved. Tell me when to reopen

I think it is now good to reopen, the culprit were those .master packages.